src/modules/navigation/components/MaterialRequestCenterBlade/MaterialRequestCenterBlade.tsx

```diff
@@ -0,0 +1,332 @@
+				<a
+					tabIndex={-1}
+					href={`/zoeken/${item.maintainerSlug}/${item.objectSchemaIdentifier}`}
+					className={styles['c-material-request-center-blade__material-link']}
+				>
+					<div
+						className={styles['c-material-request-center-blade__material']}
+						tabIndex={0}
+					>
+						<p className={styles['c-material-request-center-blade__material-label']}>
+							<Icon
+								className={
+									styles['c-material-request-center-blade__material-label-icon']
+								}
+								name={
+									item.objectType === 'audio'
+										? IconNamesLight.Audio
+										: IconNamesLight.Video
+								}
+							/>
+							<span>{item.objectSchemaName}</span>
+						</p>
+						<p className={styles['c-material-request-center-blade__material-id']}>
+							{item.objectMeemooIdentifier}
+						</p>
+					</div>
+				</a>
+				<div className={styles['c-material-request-center-blade__material-actions']}>
+					<Button
+						key={'edit-material-request'}
+						className={
+							styles['c-material-request-center-blade__material-actions-button']
+						}
+						onClick={() => editMaterialRequest(item)}
+						variants={['silver']}
+						name="Edit"
+						icon={<Icon name={IconNamesLight.Edit} aria-hidden />}
+						aria-label={tText(
+							'modules/navigation/components/material-request-center-blade/material-request-center-blade___pas-je-aanvraag-aan'
+						)}
+					/>
+					<Button
+						key={'delete-material-request'}
+						onClick={() => deleteMaterialRequest(item.id)}
+						variants={['silver']}
+						name="Delete"
+						icon={<Icon name={IconNamesLight.Trash} aria-hidden />}
+						aria-label={tText(
+							'modules/navigation/components/material-request-center-blade/material-request-center-blade___verwijder-materiaal-aanvraag'
+						)}
+					/>
+				</div>
+			</div>
+		);
+	};
+
+	const renderContent = () => {
+		if (noContent) {
+			return (
+				<p className={styles['c-material-request-center-blade__no-content']}>
+					{tText(
+						'modules/navigation/components/material-request-center-blade/material-request-center-blade___no-content'
+					)}
+				</p>
+			);
+		}
+		return (
+			mappedRequests &&
+			// Ward: render each unique maintainer
+			Object.keys(mappedRequests).map((key) => (
+				<>
+					{renderMaintainer(mappedRequests[key][0], mappedRequests[key].length)}
+
+					{/* Ward: render all materialRequests of current maintainer, sorted by objectSchemaName */}
+					{mappedRequests[key]
+						.sort((a, b) => a.objectSchemaName.localeCompare(b.objectSchemaName))
+						.map((item) => renderMaterialRequest(item))}
+				</>
+			))
+		);
+	};
+
+	const renderFooter = () => {
+		if (noContent) {
+			return (
+				<div className={styles['c-material-request-center-blade__close-button-container']}>
+					<Button
+						label={tText(
+							'modules/navigation/components/material-request-center-blade/material-request-center-blade___sluit'
+						)}
+						variants={['block', 'text']}
```

Comment:
Add `dark` variant and set the background color here instead of [here](https://github.com/viaacode/hetarchief-client/pull/697/files#diff-65a4a3170f7801ba96465c839d6bb5c5ad713b33b8f3a9641a8baf3212f2e899R122). This way we don't have to use `!important` to set the styling.

Same for the other buttons and button variants in this file/pr :)